.eslintrc.js

```diff
@@ -1,50 +0,0 @@
-/* eslint-env node */
-
-module.exports = {
-  env: {
-    es6: true,
-    node: true,
-  },
-  extends: ["eslint:recommended"],
-  globals: {
-    areInputIconsEnabled: "writable",
-    enableDataOptOut: "writable",
-    fillInputWithAlias: "writable",
-    sendRelayEvent: "readonly"
-  },
-  overrides: [
-    {
-      files: ["static/**/*.js"],
-      env: {
-        browser: true,
-      },
-      globals: {
-        ga: "readonly",
-        sendGaPing: "writable",
-        ClipboardJS: "readonly"
-      },
-    },
-  ],
-  parserOptions: {
-    ecmaVersion: 2020,
-  },
-  root: true,
-  rules: {
-    indent: ["off", 2, { SwitchCase: 1 }],
```

Comment:
Good question, which led me to check them one by one. There's a bunch of different cases, but none of them seem to require changes to our current ESLint config, I think:

- globals: :tada: no longer necessary, since we can now import modules.
- `indent`, `linebreak-style`, `quotes`, `semi` - handled automatically by Prettier
- - `no-console` - already in the recommended configs that we're extending
- `no-prototype-builtins` - no code that wants this off anymore, I think